integration/library_test.go

```diff
@@ -0,0 +1,216 @@
64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libsmartcols", Version: "2.23.2", Release: "61.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libtasn1", Version: "4.10", Release: "1.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "ca-certificates", Version: "2018.2.22", Release: "70.0.el7_5", Epoch: 0, Arch: "noarch", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "openssl-libs", Version: "1.0.2k", Release: "19.el7", Epoch: 1, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-libs", Version: "2.7.5", Release: "86.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libblkid", Version: "2.23.2", Release: "61.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "glib2", Version: "2.56.1", Release: "5.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-iniparse", Version: "0.4", Release: "9.el7", Epoch: 0, Arch: "noarch", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "cracklib", Version: "2.9.0", Release: "11.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "cracklib-dicts", Version: "2.9.0", Release: "11.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "pam", Version: "1.1.8", Release: "22.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-dateutil", Version: "1.5", Release: "7.el7", Epoch: 0, Arch: "noarch", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "libxml2-python", Version: "2.9.1", Release: "6.el7_2.3", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-gobject-base", Version: "3.22.0", Release: "1.el7_4.1", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "pkgconfig", Version: "0.27.1", Release: "4.el7", Epoch: 1, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-ethtool", Version: "0.8", Release: "8.el7", Epoch: 0, Arch: "x86_64", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-decorator", Version: "3.4.0", Release: "3.el7", Epoch: 0, Arch: "noarch", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Package{Name: "python-backports-ssl_match_hostname", Version: "3.5.0.1", Release: "1.el7", Epoch: 0, Arch: "noarch", SrcName: "", SrcVersion: "", SrcRelease: "", SrcEpoch: 0}, analyzer.Packa
```

Comment:
Is there any reason not to use `t.Run()`? It may leak `defer` unless we use `t.Run`.